readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://denys2.github.io/layout_product-cards-1/)
+- [TEST REPORT LINK](https://denys2.github.io/layout_product-cards-1/report/html_report/)
```

Comment:
These links will not work. You should fix them
```suggestion
- [DEMO LINK](https://denys2.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://denys2.github.io/layout_product-cards/report/html_report/)
```

Reply:
fix